bin/detect

```diff
@@ -5,10 +5,16 @@ set -eo pipefail
 # shellcheck disable=SC2128
 bp_dir=$(cd "$(dirname "$0")/.."; pwd)
 build_dir=$(pwd)
+build_plan="$2"
 
 # shellcheck source=/dev/null
 source "$bp_dir/lib/detect.sh"
 
 if ! detect_package_json "$build_dir" ; then
   exit 100
+else
+  cat << EOF > "$build_plan"
+  [[provides]]
+  name = "node"
+EOF
```

Comment:
Can we do the same thing here as requested in the npm buildpack: https://github.com/heroku/nodejs-npm-buildpack/pull/18#discussion_r372547386

Then we can add a unit test for it.